### ISSUE TITLE
Does it build with just released Swift major release?

### ISSUE THREAD

**Issue by etmjansen** [vapor-community/postgresql#8]
Hello,

I tried to use the Swift3 package manager to add the dependency to vapor/postgresql tag 0.3.1 using this in Package.swift

```
   dependencies: [
        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1, minor: 0),
        .Package(url: "https://github.com/vapor/postgresql.git", Version(0, 3, 1)),
    ],
```

I got the following output on 

`swift build`

```
Cloning https://github.com/vapor/postgresql.git
HEAD is now at c21260c Merge pull request #5 from vapor/swiftenv-gm-support
Resolved version: 0.3.1
error: The dependency graph could not be satisfied (https://github.com/vapor/node.git)
➜  Smartbikers cd Packages 
➜  Packages ll
total 0
drwxr-xr-x  20 topicus  staff  680 Sep 23 15:17 .
drwxr-xr-x  18 topicus  staff  612 Sep 23 15:17 ..
drwxr-xr-x  11 topicus  staff  374 Sep 23 13:00 CLibreSSL-1.0.0
drwxr-xr-x   6 topicus  staff  204 Sep 23 15:00 CPostgreSQL-0.1.1
drwxr-xr-x  11 topicus  staff  374 Sep 23 13:00 Console-1.0.0
drwxr-xr-x  11 topicus  staff  374 Sep 23 13:00 Core-1.0.0
drwxr-xr-x  10 topicus  staff  340 Sep 23 13:00 Crypto-1.0.0
drwxr-xr-x  15 topicus  staff  510 Sep 23 13:00 Engine-1.0.0
drwxr-xr-x  13 topicus  staff  442 Sep 23 13:00 Fluent-1.0.1
drwxr-xr-x  10 topicus  staff  340 Sep 23 13:00 JSON-1.0.0
drwxr-xr-x  12 topicus  staff  408 Sep 23 13:00 Jay-1.0.0
drwxr-xr-x  12 topicus  staff  408 Sep 23 13:01 Leaf-1.0.0
drwxr-xr-x  11 topicus  staff  374 Sep 23 13:00 Node-1.0.0
drwxr-xr-x  11 topicus  staff  374 Sep 23 13:00 PathIndexable-1.0.0
drwxr-xr-x  11 topicus  staff  374 Sep 23 13:00 Polymorphic-1.0.0
drwxr-xr-x  14 topicus  staff  476 Sep 23 13:00 Socks-1.0.1
drwxr-xr-x  12 topicus  staff  408 Sep 23 13:00 TLS-1.0.0
drwxr-xr-x  11 topicus  staff  374 Sep 23 13:00 Turnstile-1.0.3
drwxr-xr-x  19 topicus  staff  646 Sep 23 15:08 Vapor-1.0.2
drwxr-xr-x  12 topicus  staff  408 Sep 23 15:17 postgresql.git
```

Any suggestions?

I suppose the package manager is still buggy. I tried a simple directory rename to include the version number but that does not help. Perhaps there is a dependency conflict between current Vapor 1.2.0 and Postgresql 0.3.1 or some issue with convention of package names?

Kind regards,

Edwin Jansen

**Comment by etmjansen** [comment]
I noticed the Package.swift file on Postgresql master branch says 

`.Package(url: "https://github.com/vapor/node.git", majorVersion: 0, minor: 6)`

Latest release of Node for Swift3 is 1.0.

**Comment by etmjansen** [comment]
Seeing the swiftenv reporting 3.0-GM-CANDIDATE on build of postgresql and me being on 3.0 I think that is the source of my issues.

**Comment by etmjansen** [comment]
I rebuild with swift version 3.0 in stead of 3.0-GM-CANDIDATE and changed dependency version of vapor/node to 1.0. That builds. Now I am quite new to Swift so I am wondering how to bind this new build into my own project via the package manager. Will get back once I have learned that.

**Comment by Prince2k3** [comment]
Postgresql is not yet up to date with current releases. That is why you are getting the message. In this case Node is outdated on Postgresql while vapor is using the latest version of Node.

**Comment by natebird** [comment]
Yes, it is now updated to run on the released version of Swift 3. This was resolved in PR #9 which updated everything to the latest dependencies.
